plot-doc/plot/scribblings/renderer2d.scrbl

```diff
@@ -330,6 +332,33 @@ For example, to plot an estimated density of the triangle distribution:
 @history[#:changed "7.9" "Added support for pictures for #:label"]
 }
 
+
+@defproc[(arrows [vs  (or/c (listof (sequence/c #:min-count 2 real?))
+                            (vectorof (vector/c (sequence/c #:min-count 2 real?)
+                                                (sequence/c #:min-count 2 real?))))]
+                 [#:x-min x-min (or/c rational? #f) #f] [#:x-max x-max (or/c rational? #f) #f]
+                 [#:y-min y-min (or/c rational? #f) #f] [#:y-max y-max (or/c rational? #f) #f]
+                 [#:color color plot-color/c (arrows-color)]
+                 [#:width width (>=/c 0) (arrows-line-width)]
+                 [#:style style plot-pen-style/c (arrows-line-style)]
+                 [#:alpha alpha (real-in 0 1) (arrows-alpha)]
+                 [#:label label (or/c string? pict? #f) #f]
+                 ) renderer2d?]{
+Returns a renderer that draws either connected arrows, or couples where the first set is the arrow base and the second set is the arrow to be drawn. List and vector are interchangeble.
+For connected arrows, gaps can be introduced as shown below. Size and angle of the head are controlled by @racket[arrow-head-size-or-scale] and @racket[arrow-head-angle]
+ @interaction[#:eval plot-eval
+              (define skip '(+nan.0 +nan.0))
+              (parameterize ([arrow-head-size-or-scale '(= 20)])
+                (plot (list
+                       (arrows
+                        `((0 0) (2 1) (3 3) ,skip (0 0) (3 3))
+                        #:color 6 #:label "a=b+c")
+                       (arrows
+                        `(((2 0) (0 1)))
```

Comment:
Could you update this example to draw two vectors (add a second one), and make the second vector have a different magnitude, to illustrate this case as well.